app/index.js

```diff
@@ -76,6 +75,7 @@ var FluxGenerator = yeoman.generators.Base.extend({
     this.composeWith('flux:dispatcher', {args: ['AppDispatcher']});
     this.composeWith('flux:store', {args: ['DataStore']});
     this.composeWith('flux:action', {args: ['DataActionCreators']});
+    this.composeWith('flux:component', {args: ['App']});
```

Comment:
Ah, I see. The difference is that this generates a barebones component instead of the [more fleshed out `controller-view`](https://github.com/banderson/generator-flux-react/blob/master/app/templates/js/components/app.js) as it did previously.